src/components/overview/modals/tasks/index.tsx

```diff
@@ -0,0 +1,58 @@
+import CheckCircleIcon from '@mui/icons-material/CheckCircle';
+import { ReactComponent as CloseIcon } from '..//../../../assets/close.svg';
+import Overlay from '../../overlay';
+import RadioButtonUncheckedIcon from '@mui/icons-material/RadioButtonUnchecked';
+import React from 'react';
+import Tag from '../../tags';
+import { tasks } from '../../tasksBox/helper';
+import { BoxContainer, HeadingContainer } from './styled';
+import { Checkbox, FormControlLabel } from '@mui/material';
+import {
+  CheckboxsContainer,
+  HeadingText,
+  StatusName,
+  StyledHr,
+  ViewButton,
+} from '../../tasksBox/styled';
+
+interface TasksModalProps {
+  toggleModal: () => void;
+}
+const TasksModal = ({ toggleModal }: TasksModalProps) => {
+  return (
+    <Overlay toggleModal={toggleModal}>
+      <BoxContainer>
+        <HeadingContainer>
+          <HeadingText>Tasks</HeadingText>
+          <ViewButton onClick={toggleModal}>
+            <CloseIcon />
+          </ViewButton>
+        </HeadingContainer>
+        <StyledHr />
+        {tasks.map((task, index) => {
+          const isLastTask = index === tasks.length - 1;
+          return (
+            <React.Fragment key={index}>
```

Comment:
use just `<>` instead of `React.Fragment`

Reply:
React.Fragment is used becase it acts as a container where i can pass a key - in <></> i cannot pass key={index}.
Additionally i changed React.Fragment into just Fragment